notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -5211,6 +5211,52 @@ what it is like in a psychiatric institution [At the end of the film, a catatoni
 conformism in society [The doctors at the mental hospital were using a combination of drugs and television in an effort to make a catatonic Ira conform to societal norms.]
 
 
+movie: Decoys 2: The Second Seduction (2007)
+============================================
+
+:: Title
+Decoys 2: The Second Seduction
+
+:: Description
+Decoys 2: Alien Seduction is a 2007 Canadian science fiction horror film
+directed by Jeffery Scott Lando, and written by Miguel Tejada-Flores and
+Tom Berry. It is the sequel to the film Decoys. It was originally titled
+Decoys 2: Rebirth, with its North American DVD release title being Decoys: The
+Second Seduction.
+
+Synopsis: Grotesque aliens disguise themselves as beautiful young women in an
+effort to find human males with which to reproduce their dying kind.
+
+:: Date
+2007-03-06
+
+:: References
+https://en.wikipedia.org/wiki/Decoys_2:_Alien_Seduction
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+femme fatale [Grotesque aliens disguised as beautiful young women were going around seducing men with a view to using them as breeding stock to reproduce their dying kind. The only catch: the sex act involved is fatal for the men.]
+extraterrestrial being [Grotesque aliens disguised as beautiful young women had infiltrated a college campus in search of breeding stock.]
+what it is like at university [The story is set on a college campus and follows four male students as they hold a secret contest to see whom among them can hook up with the most college girls.]
+friendly competition in groups [Four male dorm mates held a secret contest to see whom among them can hook up with the most college girls.]
+romantic seduction [Grotesque aliens disguised as beautiful young women were going around seducing college guys before killing them in the act of human/gross-alien sex.]
+existential risk to civilization [It turned out that the aliens had come to Earth seeking breeding stock because their species was about to go extinct. No reason for why the aliens were in trouble was ever revealed.]
```

Comment:
Sounds like it was https://www.themeontology.org/theme.php?name=threat%20from%20extraterrestrial%20life or a child theme thereof

Reply:
Ah what I was attempting to convey there is that the alien civilization itself was on the brink of collapse - not that humanity was at risk of being exterminated by the breeding stock seeking alien babes/creatures.